examples/scroll-behavior/layouts/default2.vue

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
@manniL It is intentional to use `default2` as the layout name?

Reply:
@pi0 it is :relaxed:

Reply:
related to https://github.com/nuxt/nuxt.js/issues/2738#issuecomment-379489775